### PR TITLE
feat: Enable default language subpath

### PR DESCRIPTION
**Pull request by dayze**
Add of variable **defaultLocaleSubpath** to let the user choose if he wants his default language in URL
Example: If the default language is **EN**, the url will be **/en/someUrl**

From https://github.com/isaachinman/next-i18next/pull/24

### CODE REVIEW

**Comment by isaachinman** [comment]
@capellini Do you have time for a review?

**Comment by capellini** [comment]
Yes, I can find some time today for a review.

File: README.md [hunk 1]
```diff
@@ -120,6 +120,19 @@ myapp.com/de/
 myapp.com/es/
 ```
 
+If you want to enable default locale subpath, pass this option into the `NextI18Next` constructor:
```
Comment (capellini) [inline README.md]:
```suggestion
If you also want to enable locale subpaths for the default locale, pass the `defaultLocaleSubpath` option into the `NextI18Next` constructor:
```

**Comment by isaachinman** [comment]
Thanks @capellini for the review. @dayze can you check out the feedback here, and tag me when you feel things are ready? I'll then do a second review and we'll merge it if it looks good.

**Comment by dayze** [comment]
Thanks for the feedback, I'll look into it this afternoon

**Comment by dayze** [comment]
@isaachinman All good ;)

**Comment by isaachinman** [comment]
Hi @dayze - just looking through this now. Apologies for not raising this sooner, but if you had read through #24 you may have seen [this comment](https://github.com/isaachinman/next-i18next/pull/24#issuecomment-444324268)?

In short, I think it makes sense to make `localeSubpaths` an enum of two strings, as opposed to have two booleans do the same work. I can foresee a future where we might allow users to provide custom strings for locale subpaths, or might support regional locales (`en-GB`, `en-US`), and so on. This will further complicate `localeSubpaths`.

@capellini and @dayze, what do you think? I'm not too concerned with breaking changes as we're still pre-prod.

**Comment by capellini** [comment]
@isaachinman The thought had crossed my mind and I think it's a good idea, but at the time thought that it was beyond the scope of my review.  I think that would be preferable.

**Comment by isaachinman** [comment]
@dayze I think we'll just merge this, and I'll come back through shortly thereafter and update some logic. Thanks for your work.

**Comment by BondoS** [comment]
I think this option should be mentioned in README documentation.

**Comment by isaachinman** [comment]
@BondoS It is, [right here](https://github.com/isaachinman/next-i18next#5-locale-subpaths).

**Comment by BondoS** [comment]
Thank You @isaachinman, 

Yes, localeSubpath is mentioned but defaultLocaleSubpath is not.

**Comment by isaachinman** [comment]
I think you misunderstood, `defaultLocaleSubpath` is not a valid option. The `localeSubpath` option is an enum of strings.

Please read the docs carefully. Good luck!

**Pull request merged**